vmdb/product/toolbars/container_service_center_tb.yaml

```diff
@@ -16,9 +16,11 @@
       :text: 'Edit this #{ui_lookup(:table=>"container_service")}'
       :title: 'Edit this #{ui_lookup(:table=>"container_service")}'
       :url: '/edit'
+      :enabled: 'false'
     - :button: container_service_delete
       :image: delete
       :text: 'Remove this #{ui_lookup(:table=>"container_service")} from the VMDB'
       :title: 'Remove this #{ui_lookup(:table=>"container_service")} from the VMDB'
       :url_parms: '&refresh=y'
       :confirm: 'Warning: This #{ui_lookup(:table=>"container_service")} and ALL of its components will be permanently removed from the Virtual Management Database.  Are you sure you want to remove this #{ui_lookup(:table=>"container_service")}?'
+      :enabled: 'false'
```

Comment:
@h-kataria  Added to application_helper.rb the specific container types of which we want to hide the configuration button on the toolbar.  Cannot remove the yaml for these as it references those methods.  Please review